tests/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+file(GLOB TOIT_TESTS RELATIVE ${CMAKE_SOURCE_DIR} "*_test.toit" "regress/*_test.toit")
```

Comment:
Add copyright notice / license to this file.

Reply:
done.